CI.hs

```diff
@@ -135,9 +123,29 @@ parseOptions = Options
        "ghc-8.8.1" -> Right Ghc881
        "ghc-8.8.2" -> Right Ghc882
        "ghc-8.8.3" -> Right Ghc883
-       "da" -> Right Da
        "ghc-master" -> Right (GhcMaster current)
        hash -> Right (GhcMaster hash)
+   parseDaOptions :: Opts.Parser GhcFlavor
+   parseDaOptions =
+       Opts.flag' Da ( Opts.long "da" <> Opts.help "Enables DA custom build." )
```

Comment:
Maybe `da-ghc-flavor`? Something to think about. I guess this has to be exclusive with respect to `ghc-flavor`. Does the parser enforce that?

Reply:
That is my understanding of the [documentation](https://github.com/pcapriotti/optparse-applicative#alternative), and seems to be confirmed in usage:
```
$ stack runhaskell --package extra --package optparse-applicative CI.hs -- --da --ghc-flavor "ghc-8.8.1"
Invalid option `--ghc-flavor'

Usage: CI.hs (--da [--da-merge-base-sha ARG] [--da-patch ARG] 
               [--da-gen-flavor ARG] |
               --ghc-flavor ARG) [--stack-yaml ARG] [--resolver ARG] 
             [--verbosity ARG] [--cabal-verbose] [--ghc-options ARG]
  Build (and possibly upload) ghc-lib and ghc-lib-parser tarballs.
$ stack runhaskell --package extra --package optparse-applicative CI.hs -- --ghc-flavor "ghc-8.8.1" --da
Invalid option `--da'

Usage: CI.hs (--da [--da-merge-base-sha ARG] [--da-patch ARG] 
               [--da-gen-flavor ARG] |
               --ghc-flavor ARG) [--stack-yaml ARG] [--resolver ARG] 
             [--verbosity ARG] [--cabal-verbose] [--ghc-options ARG]
  Build (and possibly upload) ghc-lib and ghc-lib-parser tarballs.
$
```

Reply:
Which makes me think maybe the other flags don't need the `da` prefix? 🤔  This may be veering close to bikeshedding territory though. Happy to go with whatever you prefer for all flag names.

Reply:
If they don't need the `da` prefix, let's loose them as an elegance.

Reply:
There's no "upload" facilities any more right? Can the help comment be updated accordingly please?

Reply:
Oh right.